nservicebus/upgrades/5to6.md

```diff
@@ -632,7 +633,7 @@ snippet: 5to6-MsmqSubscriptionAuthorizer
 
 ## Synchronous request-response (callbacks)
 
-The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6. 
+The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6.
```

Comment:
> or perhaps just open and save all the files in atom to do a gobal cleanup?

that's what I'm just thinking about. Going to check whether there is an easy way to do a global cleanup.